docs/development/index.rst

```diff
@@ -33,3 +33,4 @@ setuptools changes.  You have been warned.
    developer-guide
    formats
    releases
+   history
```

Comment:
https://github.com/pypa/setuptools/pull/2425#issuecomment-709554793

Reply:
Also, there's the same problem as in  https://github.com/pypa/setuptools/pull/2426/files#r505800974